src/omv/alloc/fb_alloc.c

```diff
@@ -160,6 +170,16 @@ void *fb_alloc(uint32_t size, int hints)
         *new_pointer |= FB_OVERLAY_MEMORY_FLAG; // Add flag.
     }
     #endif
+
+    #if __DCACHE_PRESENT
+    if (hints & FB_ALLOC_CACHE_ALIGN) {
+        int offset = ((uint32_t) result) % __SCB_DCACHE_LINE_SIZE;
```

Comment:
How can offset != 0 here after aligning ?

Reply:
This is what does the alignment. The before just makes the size larger so that I ensure after this step is done that enough memory was granted.